File: modulo-00/muda-valor/mudarvalor.js
var assert = require('assert');

//
// Variáveis
//
var a, b, temp;
a = 999;
b = 555;

//
// Sua lógica
//
temp = a 
a = b  
b = temp

//
// Seu teste
//
try {
    assert.equal(555, a);
    assert.equal(999, b);
} catch(e) {
    console.log(e);
}

File: modulo-00/muda-valor/mudarvalor.c
#include <stdio.h>
#include <assert.h>

int main (){

    int a, b, temp;
    a = 999;
    b = 555;

    //
    // Sua lógica
    //
	temp = a;
	a = b;
	b = temp;
    //
    // Testes
    //
    assert(a == 555);
    assert(b == 999);

    return 0;
}